Rubberduck.Parsing/Symbols/IdentifierReferenceResolver.cs

```diff
@@ -555,13 +555,13 @@ public void Resolve(VBAParser.AsTypeClauseContext context)
             {
                 // Fixed-Length strings can have a constant-name as length that is a simple-name-expression that also has to be resolved.
                 var length = context.fieldLength();
-                if (context.fieldLength() != null)
+                if (context.fieldLength() != null && context.fieldLength().identifierValue() != null)
                 {
-                    ResolveDefault(context.fieldLength().expression());
+                    ResolveDefault(context.fieldLength().identifierValue());
```

Comment:
@autoboosh this looks good to me, the `fieldLength` can't be `const1 * const2` if `identifierValue() != null`, right?